add constraints of live migration (#284)
harvester/docs: versioned_docs/version-v1.1/vm/live-migration.md

:::note

- Live migration is not allowed when the virtual machine is using a management network of bridge interface type.

> - Live migration is not allowed when the virtual machine has any volume which belongs to `CD-ROM` type, such volume should be ejected before live migration.

Reviewer: ```suggestion
- Live migration is not allowed when the virtual machine has any volume of the `CD-ROM` type. Such volumes should be ejected before live migration.
```
